tests/foreman/cli/test_contentview.py

```diff
@@ -2201,6 +2204,59 @@ def test_positive_publish_custom_content(self):
             'Publishing new version of CV was not successful',
         )
 
+    @tier2
+    @run_only_on('sat')
```

Comment:
I think  @run_only_on('sat') is deprecated, no harm if you remove it

Reply:
@pondrejk thanks will remove that :)